Documentation/project-docs/platform-names.md

```diff
@@ -4,6 +4,24 @@ Over the years, we've accumulated quite a few names, platforms, and components t
 
 This will also list some aliases. As you'll see the aliases aren't always correct -- they are merely listed to help you find the better and less confusing terminology.
 
+In this document, the following terms are used:
+
+* **IL**. Intermediate language. IL is a stack-based instruction set and is what higher level languages, such as C#, compile down to.
```

Comment:
We should add that "IL in .NET is also sometimes called MSIL (Microsoft Intermediate Language), to separate it from other systems with intermediate languages." Or something like that.
Do we need to explain "stack-based" or is that going to far?